app/src/main/java/com/example/pocketwatching/Activities/ProfileActivity.java

```diff
@@ -52,6 +53,17 @@ public class ProfileActivity extends AppCompatActivity {
     private TextView tvTotalTokens;
     private TextView tvWelcome;
 
+    // text views that don't change
+    private TextView portfolioInformation;
+    private TextView portfolioBalance;
+    private TextView totalTokens;
+    private TextView ethAmount;
+    private TextView numTx;
+    private TextView topThreeTokens;
+    private TextView ethPrice;
+    private TextView transactionHistory;
+
+
     private static List<EthWallet> userEthWallets;
```

Comment:
Reorganize your instance variables here in the following order:
- static variables, grouping private variables and public variables
- instance variables, grouping private variables and public variables